geometry_msgs/Pose[] reference_pose
int32[] desired_config
string ur_type
bool check_q6
------------------------
ur_kinematics/JointMatrix[] solution
ur_kinematics/JointMatrix[] complete_solution
float64[] move_q6
bool success